.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug report
+description: Report to help us reproduce the bug
+labels: ['Needs Triage']
+title: "[Bug]: "
+
+body:
+- type: markdown 
+  attributes:
+    value: >
+        ## Thanks for taking the time to fill out this report
+- type: markdown 
+  attributes:
+    value: >
+       ### Before submitting a bug, please make sure it has not been reported in the 
+       [list of issues](https://github.com/narwhals-dev/narwhals/issues).
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+       A clear description of the bug.
+  validations:
+       required: true
+- type: textarea
+  attributes:
+    label: Steps or code to reproduce the bug
+    description: |
+       Please add a minimal code example to reproduce the bug.
+  validations:
+       required: true 
+- type: textarea
+  attributes:
+    label: Expected results
+    description: >
+       Please paste or describe the expected results.
+    placeholder: >
+        Example: No error is thrown.
+  validations:
+        required: true
+- type: textarea
+  attributes:
+    label: Actual results
+    description: |
+      Please paste or describe the results you observe instead of the expected results. 
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Versions
+    description: >
+      What version of Narwhals are you running?
```

Comment:
I would consider asking for pandas/polars or other dataframe library version as well.
Maybe even suggest how to find those? A naive way would probably be something like:

```terminal
pip freeze | grep -E "narwhals|polars|pandas|modin|cudf"
```

Reply:
I added the suggestion. If it still needs change, let me know please.